cloud/azure/sfx/integrations-azure.tf

```diff
@@ -23,18 +23,19 @@ resource "signalfx_org_token" "azure_integration" {
   name        = local.integration_name
   description = "Org token for ingesting data from ${local.integration_name} Azure integration"
 
-  # Where to send notifications about this token's limits. Please consult the Notification Format laid out in detectors
-  # notifications = ["Email,foo-alerts@bar.com"]
-
-  # host_or_usage_limits {
-  #   host_limit                              = 100
-  #   host_notification_threshold             = 90
-  #   container_limit                         = 200
-  #   container_notification_threshold        = 180
-  #   custom_metrics_limit                    = 1000
-  #   custom_metrics_notification_threshold   = 900
-  #   high_res_metrics_limit                  = 1000
-  #   high_res_metrics_notification_threshold = 900
-  # }
+  notifications = var.notifications_limits
+  dynamic "host_or_usage_limits" {
+    for_each = length(var.host_or_usage_limits) > 0 ? [1] : []
```

Comment:
hum, isn't it safer to check "!= null" with null by default?

Reply:
I know there was a good reason at the moment I removed null to use length but I suppose it is not true anymore